build/_verify-authenticode.shade

```diff
@@ -20,7 +20,7 @@ default expectedHash = ''
         WorkingDirectory = workingdir,
         FileName = signToolExe,
         RedirectStandardOutput = true,
-        Arguments = "verify /pa /v " + verifyFilePath,
+        Arguments = "verify /pa /v \"" + verifyFilePath + "\"",
```

Comment:
Wouldn't using single quotes for the outer strings and not escaping quite inside work?